docs/lib/geo/fragments/existing-resources.md

```diff
@@ -0,0 +1,39 @@
+You can also use Amplify Geo with your existing Amazon Location Service resources if you'd like, there's just some manual setup that you need to do.
+
+## Authorization permissions
+
+If you set up your Cognito resources manually, the roles will need to be given permission to access the map and place indices.
+
+There are two roles created by Cognito: an `Auth_Role` that grants signed-in-user-level access and an `Unauth_Role` that allows unauthenticated access to resources. Attach the corresponding policies to each role for proper access. Replace ```{account-id}``` and ```{enter Map/PlaceIndex name}``` with the correct ids and resource names.
+
+Add the following policy for both the `Auth_Role` and `Unauth_Role`:
```

Comment:
I'd say we shouldn't tell user to add to both as an instruction, rather add following policies for the appropriate roles (Auth and/or Unauth)